pkg/handlers/webhookrhobsreceiver.go

```diff
@@ -353,3 +353,39 @@ func (h *WebhookRHOBSReceiverHandler) updateManagedFleetNotificationRecord(alert
 
 	return err
 }
+
+// firingCanBeSent wraps the api's `mfnr.FiringCanBeSent`
```

Comment:
I wrapped this as I did not want to modify the API again. We should ideally have this function here, not in the API.